### ISSUE TITLE
Suggestion: provide commit count and/or number of commits since e.g. last release

### ISSUE THREAD

**Issue by GPMueller** [NOhs/git_info#15]
For example one could do something along the lines of:
```Python
    x = int(subprocess.check_output(['git', 'rev-list', '--count', 'master'], stderr=subprocess.STDOUT).decode("utf-8").strip())
    y = int(subprocess.check_output(['git', 'rev-list', '--count', branch], stderr=subprocess.STDOUT).decode("utf-8").strip())
    return y-x
```
There's plenty of potential ways to do this or what one might want to count, maybe you would find something like this useful?
In my case I use something like this this to create unique but readable package versions on any branch.

**Comment by NOhs** [comment]
You mean something like: `v1.1.3-my_branch12`?

**Comment by GPMueller** [comment]
Yes. I have other reasons, but it seems that this kind of naming is a feature of python packages on pypi, see e.g. https://github.com/pybuilder/pybuilder/issues/195.
Just a suggestion in case you would find this useful yourself.

**Comment by GPMueller** [comment]
See also [packaging.python.org](https://packaging.python.org/tutorials/distributing-packages/?highlight=distributing#standards-compliance-for-interoperability) for more info on python package numbering.
